gateway/ln-gateway/src/db.rs

```diff
@@ -1,19 +1,23 @@
+use std::collections::BTreeMap;
+
 use bitcoin::Network;
 use bitcoin_hashes::sha256;
 use fedimint_core::api::InviteCode;
 use fedimint_core::config::FederationId;
+use fedimint_core::db::{DatabaseVersion, ServerMigrationFn};
 use fedimint_core::encoding::{Decodable, Encodable};
 use fedimint_core::{impl_db_lookup, impl_db_record};
 use fedimint_ln_common::serde_routing_fees;
 use lightning_invoice::RoutingFees;
 use serde::{Deserialize, Serialize};
 use strum_macros::EnumIter;
 
+pub const GATEWAYD_DATABASE_VERSION: DatabaseVersion = DatabaseVersion(0);
+
 #[repr(u8)]
 #[derive(Clone, EnumIter, Debug)]
 pub enum DbKeyPrefix {
     FederationConfig = 0x04,
-    FederationRegistration = 0x05,
```

Comment:
This isn't used anywhere in the codebase and hasn't for sometime. I think it should be safe to remove.

Reply:
Was the key present in 0.2? If so I'd want a migration that deletes old data with that key prefix. I wouldn't bother to check probably and just add a migration, should be easy enough in this case.

Reply:
We're running 0.2.2, thanks for the call out though

Reply:
Great! Then I just misremembered :)